Gopkg.toml

```diff
@@ -79,3 +79,7 @@ ignored = ["github.com/improbable-eng/thanos/benchmark/*"]
 [[constraint]]
   name = "github.com/Azure/azure-storage-blob-go"
   revision = "197d1c0aea1b9eedbbaee0a1a32bf81e879bde80"
+
+[[constraint]]
+  name = "github.com/sbueringer/thanos-inspect"
```

Comment:
Can we avoid dependency on tooling package?  Especially that we decided to embed this nice code into our project, but this PR is kind of "in between" (:  We embeded something but left something. 

I would say we either reference this tool in docs OR remove this as dependency. Each dependency adds maintainance burden. Especially if we do it only for small 100 LOC `table` package.

Reply:
I integrated the required code into bucket.go.